diffsims/generators/rotation_list_generators.py

```diff
@@ -159,83 +161,521 @@ def get_grid_around_beam_direction(beam_rotation, resolution, angular_range=(0,
     return rotation_list
 
 
-def get_beam_directions_grid(crystal_system, resolution, equal="angle"):
+def normalize_vectors(vectors):
```

Comment:
This all looks very nice.

Just a quick question: have you considered using `orix.vector.Vector3d` to store vectors (https://orix.readthedocs.io/en/stable/reference.html#module-orix.vector.vector3d)? If `v` is a vector, you can use `v.unit` to get the normalized vectors.

Reply:
Didn't consider it mainly because I wasn't aware of this documentation and because I didn't want to delve too deeply into what is already available in orix. Also, since orix is still changing a lot, it seems safer to implement basic functionality like these meshing functions with lower level things like numpy arrays. I'm not too keen now to look for all the places where I could have used orix, but if you feel like it will improve the maintainability of the code in the long run feel free to change it.

Reply:
Sounds reasonable, so I wouldn't consider it for this PR.